src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -126,19 +126,19 @@ static void move_research_item(ResearchItem* beforeItem, int32_t scrollIndex);
 static void research_rides_setup()
 {
     // Reset all objects to not required
-    for (uint8_t objectType = OBJECT_TYPE_RIDE; objectType < OBJECT_TYPE_COUNT; objectType++)
+    for (uint8_t objectType = EnumValue(ObjectType::Ride); objectType < EnumValue(ObjectType::Count); objectType++)
```

Comment:
Would it be possible to change the type of `objectType` to `ObjectType` instead of `uint8_t`?